sympy/concrete/summations.py

```diff
@@ -473,6 +474,12 @@ def is_convergent(self):
         if not dict_val[p].has(sym) and is_decreasing(dict_val[q], interval):
             return S.true
 
+        ### ------------- Limit comparison test -----------###
+        # (1/n) comparison
+        lim_comp = limit(sym*sequence_term, sym, oo)
```

Comment:
Could `upper_limit` be used here instead of `oo` that needs to be imported.

Reply:
But the test specifically mentions the `upper_limit` to be infinity. I think it will be better to keep it that way, keeping the syntax of the test intact.
@jksuom

Reply:
I think that all sums at this point actually have limit `oo`. (Otherwise, the sum is already known to converge.) But maybe `S.Infinity` could be used as `S` has been imported.

Reply:
Changed `oo` to `S.Infinity`. @jksuom